PayPalWebCheckout/src/main/java/com/paypal/android/checkoutweb/PayPalWebCheckoutFundingSource.kt

```diff
@@ -4,5 +4,18 @@ package com.paypal.android.checkoutweb
  * Enum class to specify the type of funding for an order
  */
 enum class PayPalWebCheckoutFundingSource(val value: String) {
-    CREDIT("credit"), PAY_LATER("paylater"), PAYPAL("paypal")
+    /**
+     * CREDIT will launch the web checkout flow with credit funding selected
+     */
+    CREDIT("credit"),
+
+    /**
+     * PAY_LATER will launch the web checkout flow with pay later selected
+     */
+    PAY_LATER("paylater"),
+
+    /**
+     * PAYPAL will launch the web checkout default flow
```

Comment:
```suggestion
     * PAYPAL will launch the web checkout for a one-time PayPal Checkout flow.
```

Is this accurate? Do we only offer one time checkout through the default or is vaulting possible here too?

Reply:
Im not sure. Passing no value for the `fundingSource` query param, lanuches the same flow as passing `paypal`, so im guessing its just the default. I do have to log in every time.